config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml

```diff
@@ -21,7 +21,7 @@ periodics:
           pushd "$ARTIFACTS"; \
           go install github.com/kubernetes-sigs/depstat@latest; \
           popd; \
-          depstat stats --json || true; 
```

Comment:
do you really need `|| true`? is the depstat command expected to fail, i'd rather see RED when it does.